scripts/ASTImportTest.sh

```diff
@@ -131,6 +146,125 @@ function test_ast_import_export_equivalence
     TESTED=$((TESTED + 1))
 }
 
+function test_evmjson_import_export_equivalence
+{
+    local sol_file="$1"
+    local input_files=( "${@:2}" )
+    local outputs=( "asm" "bin" "bin-runtime" "opcodes" "srcmap" "srcmap-runtime" )
+    local export_command=("$SOLC" --combined-json "$(IFS=, ; echo "${outputs[*]}")" --pretty-json --json-indent 4 "${input_files[@]}")
+    local success=1
+    if ! "${export_command[@]}" > expected.json 2> expected.error
+    then
+        success=0
+        printError "ERROR: (export) EVM Assembly JSON reimport failed for ${sol_file}"
+        if [[ $EXIT_ON_ERROR == 1 ]]
+        then
+            print_used_commands "$(pwd)" "${export_command[*]}" ""
+            return 1
+        fi
+    fi
+    if ! "${export_command[@]}" "--optimize" > expected.optimize.json 2> expected.optimize.error
+    then
+        success=0
+        printError "ERROR: (export with --optimize) EVM Assembly JSON reimport failed for ${sol_file}"
+        if [[ $EXIT_ON_ERROR == 1 ]]
+        then
+            print_used_commands "$(pwd)" "${export_command[*]} --optimize" ""
+            return 1
+        fi
+    fi
+
+    for input_json in "expected.json" "expected.optimize.json"
+    do
+        local optimize_param=""
+        if [[ "$input_json" == "expected.optimize.json" ]]
+        then
+            optimize_param="--optimize"
+        fi
+
+        # Note that we have some test files, that only consists of free functions.
+        # Those files doesn't define any contracts, so the resulting json does not define any
+        # keys. In this case `jq` returns an error like `jq: error: null (null) has no keys`
+        # to not get spammed by these errors, errors are redirected to /dev/null.
```

Comment:
Also, if this is the only situation that makes it print errors, I think it would be better to add a check just specifically for this condition. I.e. skip the file when `[[ $(jq 'keys' "$input_json") == "[]" ]]`. This would avoid silencing other errors that might happen here.